.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "divlo.vscode-styled-jsx-languageserver",
```

Comment:
no need for this (and the one below) in this codebase

Reply:
Removing in the next PR!